server/tools/hatohol-db-initiator.in

```diff
@@ -132,6 +132,11 @@ def parse_default_conf():
 
 def get_default_conf_item(item_name):
     config = parse_default_conf()
+    if config is None:
+       global not_found_config_file
+       not_found_config_file = True
+       return "<Unknown>"
```

Comment:
ashie-san

How's that for db_name, db_user, and db_password ?
The previous version needs these parameters as a command line option. So there's no default value.